### PR TITLE
Issue #1163 - Convert PublicDNS to use template

### PR DESCRIPTION
**Pull request by Mailkov**
-----
IA Page: https://duck.co/ia/view/public_dns

### CODE REVIEW

File: share/goodie/public_dns/content.handlebars [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<div class="gw">
+		<div class="g  third"><a href='{{info_url}}'>{{provider}}</a></div>
```
Comment (moollaza) [inline share/goodie/public_dns/content.handlebars]: Looks like the indentation is a little off here?

Comment (Mailkov) [inline share/goodie/public_dns/content.handlebars]: @moollaza Yes, but is strange I don't use tab

**Comment by daxtheduck** [comment]
Automated data from [IA page](https://duck.co/ia/view/public_dns)

---
**Basic Info**

**Description**: list common public DNS servers and their IP addresses
**Example Query**: public dns
**Tab Name**: answer
**Source**: 

---
**Testing**

**Browsers**
[ ] safari
[ ] firefox
[ ] ie
[ ] opera
[ ] chrome


**Mobile**
[ ] android
[ ] ios

**Comment by moollaza** [comment]
@Mailkov this is looking good. I fiddled with it a bit and arrived at this:

```handlebars
<div class="gw">
    <div class="g third">
		<a class="text--primary" href='{{info_url}}'>{{provider}}</a>
	</div>

	<small class="g third">
        {{#each ip4}}
            <div>{{this}}</div>
        {{/each}}
    </small>

	{{#if ip6}}
    <small class="g third">
        {{#each ip6}}
            <div>{{this}}</div>
        {{/each}}
    </small>
	{{/if}}
</div>
```

The result looks like this:
public_dns_at_duckduckgo

I'm trying to avoid a separate CSS file, but I don't love the top-aligned DNS Server names. I think for now this is OK.

**Comment by moollaza** [comment]
@Mailkov can you please fix the merge conflict here? Otherwise this LGTM!

**Comment by Mailkov** [comment]
@moollaza merge conflict fixes

**Comment by moollaza** [comment]
@Mailkov Thanks! This LGTM :+1:

**Pull request merged**